react/src/components/Products.js

```diff
@@ -66,7 +66,16 @@ class Products extends Component {
     try {
       products = await this.getProducts();
       // take first 4 products because that's all we have img/title/description for
-      this.props.setProducts(products.slice(0, 4));
+      this.props.setProducts(
+        // round to nearest int so we can adjust w/o having to worry about floats
+        Array(Math.round(200 / 4))
```

Comment:
Isn't immediately obvious why we're doing `200/4` rather than `50`. Is this because we have 4 products? If so, consider making a `NUM_PRODUCTS` constant and pulling those references to `4` out into that for readability